### ISSUE TITLE
no srcset generated

### ISSUE THREAD

**Issue by silveltman** [divriots/jampack#82]
My index.astro page:
```
---
import Layout from '@layouts/Layout.astro'
---

<Layout>
  <img src="/hero.jpg" alt="myimg">
</Layout>
```
Gets generated into:
```
<img alt="myimg" decoding="async" fetchpriority="high" height="1165" src="/hero.jpg" width="1427">
```

I can't find why, no errors warning nothing

Console output:
```
$ jampack ./dist
     __                                    __    
    |__|____    _____ ___________    ____ |  | __
    |  \__  \  /     \\____ \__  \ _/ ___\|  |/ /
    |  |/ __ \|  Y Y  \  |_> > __ \\  \___|    < 
/\__|  (____  /__|_|  /   __(____  /\___  >__|_ \
\______|    \/      \/|  |       \/     \/     \/
 v0.13.0              |__| by ‹div›RIOTS

Options:
{}

Overriding default config with:
{
  "image": {
    "compress": false,
    "external": {
      "process": "download"
    }
  }
}

 PASS 1 - Optimizing 
▶ index.html
  ✔ <img> [2/2]
  ✔ <iframe> [2/2]
▶ algemene-voorwaarden/index.html
  ✔ <img> [0/0]
  ✔ <iframe> [0/0]
▶ index/index.html
  ✔ <img> [1/1]
  ✔ <iframe> [1/1]
▶ privacyverklaring/index.html
  ✔ <img> [0/0]
  ✔ <iframe> [0/0]
Done: 44.087ms

 PASS 2 - Compressing the rest 
✔ 21 files | 2.92 MB → 2.41 MB | -523.21 KB 
Done: 26.618ms

 Summary 
╔════════╤════════════╤═══════════╤════════════╤════════════╗
║ Action │ Compressed │  Original │ Compressed │       Gain ║
╟────────┼────────────┼───────────┼────────────┼────────────╢
║ .jpg   │      1 / 4 │   2.18 MB │    1.82 MB │ -363.13 KB ║
║ .css   │      2 / 2 │  42.52 KB │   39.14 KB │   -3.38 KB ║
║ .js    │      3 / 3 │  36.37 KB │   35.81 KB │  -569.00 B ║
║ .html  │      4 / 4 │  11.28 KB │   11.25 KB │   -28.00 B ║
║ .webp  │      3 / 3 │ 601.57 KB │  481.31 KB │ -120.26 KB ║
║ .png   │      1 / 1 │  51.50 KB │   15.64 KB │  -35.86 KB ║
║ .txt   │      0 / 1 │   73.00 B │    73.00 B │            ║
║ .xml   │      0 / 2 │  744.00 B │   744.00 B │            ║
║ .svg   │      0 / 1 │  11.20 KB │   11.20 KB │            ║
╟────────┼────────────┼───────────┼────────────┼────────────╢
║ Total  │    14 / 21 │   2.92 MB │    2.41 MB │ -523.21 KB ║
╚════════╧════════════╧═══════════╧════════════╧════════════╝


 ✔ No issues 
✨  Done in 3.51s.
```

**Comment by silveltman** [comment]
I found that after removing my jampack.config.js file it does work. This was my config:
```
export default {
    image: {
      compress: false,
      external: {
        process: 'download',
      }
    },
  };
```

**Comment by georges-gomes** [comment]
Hi @silveltman, it should merge so you probably found a bug.

**Comment by georges-gomes** [comment]
And the bug is that with `compress: false` no srcset is created.
But I get you don't want to compress originals but still want srcset images to be created.
I will fix that

**Comment by silveltman** [comment]
In that case, I think the console output should be modified too.
It says `Overriding default config with:`, which I think should be `Merging default config with:'

**Comment by georges-gomes** [comment]
Fixed both compress=false and srcsets + console output is now `Overriding default config with:`
Thanks!

Available in `0.14.1+` release to be done
